### ISSUE TITLE
Termes employés

### ISSUE THREAD

**Issue by RLesur** [strainel/copyre-alpha#4]
En relisant le terme "data-codeurs", j'ai réalisé que cette expression serait susceptible de faire peur à pas mal de monde chez moi (j'ai des B et C qui sont intéressés pour apprendre, découvrir... mais qui ne se reconnaîtront jamais dans une telle expression).
Je me demande si on devrait pas faire attention à employer des termes les plus inclusifs possible.

Qu'en pensez-vous ?

**Comment by strainel** [comment]
oui je suis d'accord ; il faut faire attention sur les termes.

**Comment by Welsh44** [comment]
Exact. Cela peut être l'occasion de faire découvrir des technos, des façons de faire à des agents qui sont (parfois) un peu loin de cela.

**Comment by strainel** [comment]
je voulais transférer l'issue vers le nouveau projet... pas moyen :)
